lib/surface/components/form/label.ex

```diff
@@ -23,6 +26,11 @@ defmodule Surface.Components.Form.Label do
   @doc "The CSS class for the underlying tag"
   prop class, :css_class
 
+  @doc """
+  The text for the generated `<label>` element, if no content (default slot) is provided.
+  """
+  prop text, :any
```

Comment:
should this be a `:string` with `default: nil`?

Reply:
TIL :)